client/state/test/utils.js

```diff
@@ -235,4 +244,74 @@ describe( 'utils', () => {
 			expect( state ).to.eql( [ 0, 1 ] );
 		} );
 	} );
+
+	describe( '#keyedReducer', () => {
+		const grow = name => ( { type: 'GROW', name } );
+
+		const age = ( state = 0, action ) =>
+			'GROW' === action.type
+				? state + 1
+				: state;
+
+		const prevState = {
```

Comment:
might consider `deepFreeze` here to prevent tests from mutating the previous state